### DISCUSSION TITLE
A questing about the select component

### DISCUSSION THREAD

**Discussion by krenarxh** [tasteui/tasteui#577] [Q&A]
I want to put a selector for the user model, it works well, but when you have 100+ users it lags a lot, what I wanted to do is to show only a limited number of user names in the selection and to use the search function to search for the other users, is it anyway to do this with the current component?

**Comment by devajmeireles** [comment]
The best thing to do is adapt the select to have a lazy loading option, I'm not sure about going ahead and adding that, I'll sleep on it. For now, I suggest you use the API option, where the search can be limited according to your input in the search box.

**Reply by krenarxh** [reply]
I will try that, thank you.
